nodebuilder/rpc/config.go

```diff
@@ -20,6 +29,16 @@ func DefaultConfig() Config {
 		// do NOT expose the same port as celestia-core by default so that both can run on the same machine
 		Port:     defaultPort,
 		SkipAuth: false,
+		CORS:     DefaultCORSConfig(),
+	}
+}
+
+func DefaultCORSConfig() CORSConfig {
+	return CORSConfig{
+		Enabled:        false,
+		AllowedOrigins: []string{},
+		AllowedHeaders: []string{},
+		AllowedMethods: []string{},
```

Comment:
Seems to miss defaults from the task

Reply:
The default configuration for CORS is that CORS is disabled. So the other parameters are set to empty as the default values.

The case where cors is enabled without a possible subsetting, i.e., `--rpc.cors-allowed-origins` or `--rpc.cors-allowed-methods` or `--rpc.cors-allowed-headers`  , in that case, default values are filled in while parsing the flags for each of the subsetting. 

Do you think this should be moved someplace outside the `ParseFlags` in `nodebuilder/rpc/flags.go/` ?